File: iOS/Studien/Studien/Stroke.m
//
//  Stroke.m
//  Studien
//
//  Created by Odie Edo-Osagie on 21/07/2016.
//  Copyright © 2016 Odie Edo-Osagie. All rights reserved.
//

#import "Stroke.h"

@implementation Stroke

- (id) initWithStartPoint:(WBPoint *)point andColor:(UIColor *)color
{
    self = [super init];
    
    if(self){
        _points = [NSMutableArray arrayWithObject:point];
        _color = color;
    }
    
    return self;
}

- (void) addPoint:(WBPoint *)point
{
    [_points addObject:point];
}

- (WBPoint *) startPoint
{
    return _points[0];
}

@end

File: Android/Studien/app/src/main/java/com/odie/studien/Models/Stroke.java
package com.odie.studien.Models;

import java.util.ArrayList;

/**
 * Created by Odie on 17/09/2016.
 */
public class Stroke {
    public ArrayList<WBPoint> points;

    public Stroke(){
        this.points = new ArrayList<>();
    }

    public Stroke(ArrayList<WBPoint> pointList){
        this.points = pointList;
    }

    public void addPoint(WBPoint p){
        points.add(p);
    }
}
